linchpin/provision/roles/libvirt/files/schema.json

```diff
@@ -17,6 +17,7 @@
                     "memory": { "type": "integer", "required": true },
                     "count": { "type": "integer", "required": false },
                     "uri": { "type": "string", "required": false },
+                    "remote_python_interpretor": { "type": "string", "required": false },
```

Comment:
nit: spelling - `interpreter`

Reply:
@samvarankashyap

Reply:
@samvarankashyap There's still one more place where this is spelled incorrectly.  Aside from the spelling error, it'll cause `tasks/provision_libvirt_node.yml` to fail because its spelled correctly there